src/components/icon/mono-icon.tsx

```diff
@@ -19,18 +20,31 @@ export type MonoIconProps = Omit<JSX.IntrinsicElements['img'], 'src'> & {
    * @default none
    */
   overrideMode?: 'none' | 'dark' | 'light';
+  /**
+   * Used to override light- or dark mode icon color when MonoIcon is used with an interactive component
+   */
+  interactiveColor?: keyof Theme['interactive'];
```

Comment:
Are `interactiveColor` and `interactiveState` both going to be present at the same time?

Reply:
It can be. There is no need for the `interactiveState` if you don't use the `interactiveColor`, but you can use the `interactiveColor` without the `interactiveState`. Most likely you would use both at the same time to handle hover/disabled colors etc.